### PR TITLE
my history commit

### PR DESCRIPTION
**Pull request by 04goat**
hello

### CODE REVIEW

**Comment by JohnnyTTran** [comment]
He ain't going to accept your req LMAO

**Comment by jerdoa** [comment]
Hello

File: bryan/bryan_history.txt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+   45  ls
+   46  vim index.html
+   47  cat hello world, Bryan
+   48  open index.html
+   49  ls
+   50  git checkout -b first_branch
+   51  git status
+   52  git add index.html
+   53  git status
+   54  git commit - m "this is my first commit"
```
Comment (ReubenNahouraii) [inline bryan/bryan_history.txt:54]: it looks like you figured it out... it is -m not - m (the m stands for message). All command flags look like this, ie, -flag. Does this make sense?

Comment (04goat) [reply bryan/bryan_history.txt:54]: I might not remember all of that, but I'll keep an eye on that. Thank you.

**Pull request merged**